src/main/java/uk/ac/ebi/eva/evaseqcol/controller/seqcol/SeqColController.java

```diff
@@ -119,10 +122,12 @@ public ResponseEntity<?> getAttribute(@Parameter(name = "attribute",
                                                   example = "3mTg0tAA3PS-R1TzelLVWJ2ilUzoWfVq",
```

Comment:
Previous line (description) - should this be "Attribute's level 1 digest"? (not sure about the exact terms)

Reply:
corrected